src/cargo/core/resolver/features.rs

```diff
@@ -812,7 +812,7 @@ impl<'a, 'cfg> FeatureResolver<'a, 'cfg> {
                         true
                     })
                     .flat_map(|dep| {
-                        // Each `dep`endency can be built for multiple targets. For one, it
+                        // Each dependency can be built for multiple targets. For one, it
```

Comment:
This is unrelated and the old version was trying to highlight it is referring to the closures variable.